pom.xml

```diff
@@ -73,16 +73,17 @@
     </licenses>
 
     <scm>
-        <developerConnection>scm:git:https://github.com/citerus/dddsample-core</developerConnection>
-        <url>https://github.com/citerus/dddsample-core</url>
+        <developerConnection>scm:git:https://github.com/aowss/dddsample-core</developerConnection>
```

Comment:
This line should stay as `https://github.com/citerus/dddsample-core`.

Reply:
of course